docs/deployment/cli-flags.md

```diff
@@ -0,0 +1,34 @@
+# CLI Flags
+
+Etcd-druid exposes the following CLI flags that allow for configuring its behavior.
+
+| CLI FLag                                | Component                        | Description                                                                                                                                                                                                                                                                                                     | Default                        |
+|-----------------------------------------|----------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------|
+| `feature-gates`                         | `etcd-druid`                     | A set of key=value pairs that describe feature gates for alpha/experimental features. Please check [feature-gates](feature-gates.md) for more information.                                                                                                                                                      | `""`                           |
```

Comment:
Feature gates are for the entire druid, defined in https://github.com/unmarshall/etcd-druid/blob/refactor-ctrls/internal/features/features.go. Within druid, each controller/webhook can have its own `featureList`, which specifies which of these features are relevant to it. Currently, the only feature gate in druid is `UseEtcdWrapper`, which is relevant for etcd, etcdcopybackupstask and compaction controllers, since they all need to know which image to use when creating their respective sts/job objects.

Hence, `--feature-gates` is for druid as a whole, and then how it is used is up to each controller to decide for itself.